src/main/rx/msp.h

```diff
@@ -22,5 +22,7 @@
 
 struct rxConfig_s;
 struct rxRuntimeState_s;
+uint16_t rxMspReadRawRC(const rxRuntimeState_t *rxRuntimeState, uint8_t chan);
+uint8_t rxMspFrameStatus(rxRuntimeState_t *rxRuntimeState);
```

Comment:
I think this is not needed any more now.

Reply:
Ah, yes that is true. I removed the `rxMspOverrideFramStatus`, but forgot to hide this one